foundationdb-gen/src/bin/foundationdb-options-gen.rs

```diff
@@ -111,7 +111,7 @@ impl FdbScope {
 
         let mut s = String::new();
         s += &format!(
-            "pub unsafe fn apply(&self{}) -> std::result::Result<(), error::FdbError> {{\n",
+            "pub unsafe fn apply(&self{}) -> std::result::Result<(), error::Error> {{\n",
```

Comment:
while this crate is tested, is there and integration test for the resultant output of this generator? might be a gap in our testing.